client/src/tokens/colors.js

```diff
@@ -159,7 +159,7 @@ const colors = {
   },
   warning: {
     100: {
-      hex: '#FFBF00',
+      hex: '#FAA500',
```

Comment:
After design review with Ben we decided to modify the hue of the warning color. Currently this isn't used anywhere either.